src/style.css

```diff
@@ -0,0 +1,48 @@
+
+body {
+  /* disable text higlightening */
+  -webkit-touch-callout: none;
+  /* iOS Safari */
+  -webkit-user-select: none;
+  /* Safari */
+  -khtml-user-select: none;
+  /* Konqueror HTML */
+  -moz-user-select: none;
+  /* Firefox */
+  -ms-user-select: none;
+  /* Internet Explorer/Edge */
+  user-select: none;
+  /* Non-prefixed version, currently supported by Chrome and Opera */
+  margin: auto;
+}
+
+/* disable spinner in the number input field */
```

Comment:
Isn't this supposed to move to before line 28?

Reply:
No, all of these lines are part of this style.

Reply:
Oh ok